endpointmanager/cmd/datavalidation/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"math"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/config"
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/helpers"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager/postgresql"
+)
+
+func main() {
+	err := config.SetupConfig()
+	helpers.FailOnError("", err)
+
+	store, err := postgresql.NewStore(viper.GetString("dbhost"), viper.GetInt("dbport"), viper.GetString("dbuser"), viper.GetString("dbpassword"), viper.GetString("dbname"), viper.GetString("dbsslmode"))
+	helpers.FailOnError("", err)
+	log.Info("Running data validation check")
+
+	queryInterval := viper.GetInt("capquery_qryintvl")
+	maxEndpoints := int(math.Round(float64(queryInterval*60) / float64(1.5)))
```

Comment:
I think there should be a comment here explaining the average 1.5 second per request metric